src/app.js

```diff
@@ -0,0 +1,178 @@
+                } if (res.ok === true && lengthProcess === 5) {
+                    let result = { message: `${uri} ${link} work ${res.status} total: ${totalLinks}` }
+                    result.working = "work"
+                    statsLinks(result, totalLinks, arrContent)
+                } else {
+                    let result = {}
+                    result.href = `${link.slice(0, 50)}`,
+                        result.status = `${res.status} ok`,
+                        result.file = `${route}`
+                    resolve(result)
+                }
+            })
+            .catch(err => {
+                if (lengthProcess === 4) {
+                    let result = { file: `${uri}` }
+                    result.href = ` ${link}`
+                    result.status = "400 is broken"
+                    console.log(chalk` {rgb(255,89,94) ✖} ${uri} ${link.slice(0, 50)} {rgb(255,89,94) is broken}`);
+                    resolve(result)
+                }
+                if (lengthProcess === 5) {
+                    let result = { message: `${uri} ${link} is broken`, working: "is broken" }
+                    statsLinks(result, totalLinks, arrContent)
+                } else {
+                    let result = {}
+                    result.href = ` ${link.slice(0, 50)}`
+                    result.status = "400 is broken"
+                    resolve(result)
+                }
+            })
+    }))
+    return Promise.all(arr)
+
+}
+
+
+let newArr = []
+let counter = 0
+function statsLinks(result, totalLinks, arrContent) {
+    let totalArr = newArr.length + 1
+    newArr.push(result.working)
+    if (totalLinks === totalArr) {
+        let filtBroke = newArr.filter(news => news === 'is broken')
+        counter = counter + filtBroke.length
+        let totalBasic = arrContent.length
+        let uniqueLinks = arrContent.filter(unique);
+        let howManyUnique = uniqueLinks.length
+
+        function unique(value, index, self) {
+            return self.indexOf(value) === index;
+        }
+        console.log(chalk.italic('Validation statistics:'));
+        console.log(chalk`{bold Total:} {cyan ${totalBasic}}`);
+        console.log(chalk`{bold Unique:} {cyan ${howManyUnique}}`);
+        console.log(chalk`{bold Broken:} {cyan ${counter}}`);
+        let objStatus = { Total: totalBasic, Unique: howManyUnique, Broken: counter }
+        return objStatus
+    }
+
+}
+
+
+function statsBasic(arrLinks) {
+    let totalBasic = arrLinks.length
+    console.log(chalk.italic('Basic statistics:'));
+    console.log(chalk`{bold Total:} {cyan ${totalBasic}}`);
+    let uniqueLinks = arrLinks.filter(unique);
+    let howManyUnique = uniqueLinks.length
+    console.log(chalk`{bold Unique:} {cyan ${howManyUnique}}`);
+
+    function unique(value, index, self) {//comprueba, si el valor dado es el primero que ocurre, de lo contrario, debe ser un duplicado y no se copiará.
```

Comment:
Tus comentarios para documentar tu código deben de estar arriba 
```suggestion
//comprueba, si el valor dado es el primero que ocurre, de lo contrario, debe ser un duplicado y no se copiará.
    function unique(value, index, self) {
```